cpp/src/examples/union_example.cpp

```diff
@@ -22,9 +22,13 @@
  * This example reads two csv files and does a union on them.
  */
 int main(int argc, char *argv[]) {
-  if (argc < 3) {
-    LOG(ERROR) << "There should be two arguments with paths to csv files";
-    return 1;
+  std::string path1, path2;
+  if (argc != 3) {
+    path1 = "/tmp/csv1.csv";
+    path2 = "/tmp/csv2.csv";
```

Comment:
I think this can be confusing to the user as we are expecting two files with fixed names in the /tmp folder

Reply:
(y)